src/modules/book-favorite/book-favorite.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common';
+import { BookFavoriteService } from './book-favorite.service';
+import { BookFavoriteItemDto } from 'src/modules/book-favorite/dto/book-favorite-item.dto';
+import { BookFavoriteCreateDto } from 'src/modules/book-favorite/dto/book-favorite-create.dto';
+import type { Uuid } from 'src/common/type';
+
+@Controller('book-favorites')
+export class BookFavoriteController {
+  constructor(private readonly bookFavoriteService: BookFavoriteService) {}
+
+  @Get()
+  async findAll(): Promise<BookFavoriteItemDto[]> {
+    return BookFavoriteItemDto.fromBookFavorites(
+      await this.bookFavoriteService.findAll(),
+    );
+  }
+
+  @Post()
+  async create(
+    @Body() bookFavoriteCreateDto: BookFavoriteCreateDto,
+  ): Promise<BookFavoriteItemDto> {
+    return BookFavoriteItemDto.fromBookFavorite(
+      await this.bookFavoriteService.create(
+        BookFavoriteCreateDto.toBookFavoriteCreate(bookFavoriteCreateDto),
+      ),
+    );
+  }
+
+  @Delete(':id')
+  async remove(@Param('id') id: Uuid): Promise<void> {
```

Comment:
Frontend không cần biết id của book favorite, ở đây có thể là id của book

```suggestion
  @Delete(':bookId')
  async remove(@Param('id') bookId: Uuid): Promise<void> {
```